### ISSUE TITLE
5.2 support?

### ISSUE THREAD

**Issue by aaronhdgolf** [RLoris/SocketHelperDoc#7]
Are you going to update to 5.2?

**Comment by RLoris** [comment]
Yeah I have many plugins, I already started the update process to 5.2 for 4 of them, I will submit the rest this weekend when I have some free time, epic will take some time to approve updates anyway
